common/src/app/common/types/text.cljc

```diff
@@ -66,7 +102,7 @@
    entries"
   [a b]
   (cond
-    (not= (:type a) (:type b))
+    (not= (type a) (type b))
```

Comment:
I'm not clearly understand this change, are you trying to check if two value has different runtime type? is that correct?